docs/DESIGN_TOKENS.md

```
@@ -0,0 +1,288 @@
+@import "../styles/design-system.css";  /* Single source of truth */
+@import "../styles/theme.css";          /* Feature accents & animations only */
+```
+
+**Why this matters:**
+- `design-system.css` contains all base design tokens (colors, spacing, shadows, etc.)
+- `theme.css` contains only feature-specific accents and animation keyframes
+- Later imports can override earlier ones, so order is critical
+
+### File Responsibilities
+
+#### `design-system.css` - Base Token System
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)